scripts/map.js

```diff
@@ -532,8 +529,6 @@ var loadState = function () {
         fillPattern: stripes,
       },
     })
-
-    overlays['state'].addTo(map)
```

Comment:
Redundant with the layer being added via the input on line 127? 
Prob copy/pasta gone awry 😸 
Ninja catch on your part 🥷